### PR TITLE
genkeys support on Windows

### PR DESCRIPTION
**Pull request by aminya**

### CODE REVIEW

File: src/genkeys.jl [hunk 1]
```diff
@@ -56,9 +56,16 @@ LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBNnpiRkdXQVZpYlIy
 ```
 """
 function genkeys(; user="\$USER", repo="\$REPO")
-    # Error checking. Do the required programs exist?
-    success(`which which`)      || error("'which' not found.")
-    success(`which ssh-keygen`) || error("'ssh-keygen' not found.")
+    if Sys.iswindows()
+        # Error checking. Do the required programs exist?
+        success(`where where`)      || error("'where' not found.")
+        success(`where git`)        || error("'git' not found.")
```
Comment (mortenpi) [inline src/genkeys.jl]: This should be `success(`which ssh-keygen`) || error("'ssh-keygen' not found.")`, not Git?

Comment (aminya) [reply src/genkeys.jl]: Fixed in https://github.com/JuliaDocs/DocumenterTools.jl/pull/33/commits/49ab299bce8824ff1139fa665c8e72ebb27ae2fa

Comment (mortenpi) [reply src/genkeys.jl]: e30962b modified the wrong `genkeys` method I'm afraid.

Comment (aminya) [reply src/genkeys.jl]: No, I check the existence of ssh-kegen both on Linux and Windows.
The tests pass if you see.

Comment (mortenpi) [reply src/genkeys.jl]: This method does not use `git` though, so we shouldn't require it to be present here.

```suggestion
```

File: src/genkeys.jl [hunk 2]
```diff
@@ -57,8 +57,16 @@ LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBNnpiRkdXQVZpYlIy
...
-    success(`which which`)      || error("'which' not found.")
-    success(`which ssh-keygen`) || error("'ssh-keygen' not found.")
+    if Sys.iswindows()
+        success(`where where`)      || error("'where' not found.")
+        success(`where ssh-keygen`) || error("'ssh-keygen' not found.")
+        success(`where git`)        || error("'git' not found.")
+    else
+        success(`which which`)      || error("'which' not found.")
+        success(`which ssh-keygen`) || error("'ssh-keygen' not found.")
+        success(`which git`)        || error("'git' not found.")
```
Comment (mortenpi) [inline src/genkeys.jl]:
```suggestion
```

File: src/genkeys.jl (hunk 1 above)
Comment (aminya) [reply src/genkeys.jl]: done in https://github.com/JuliaDocs/DocumenterTools.jl/pull/33/commits/3fa8676703dc19227b8fe993520e4b26a4d4afad

**Comment by mortenpi** [comment]
LGTM now, thanks!

**Pull request merged**